src/store/Transaction.ts

```diff
@@ -145,8 +199,20 @@ export default {
         ) => {
             state.partialTransactions = conditionalSort(transactions, transactionComparator);
         },
-        setDisplayedTransactionStatus: (state: TransactionState, displayedTransactionStatus: TransactionGroupState) => {
-            state.displayedTransactionStatus = displayedTransactionStatus;
+        setAllTransactions: (state: TransactionState) => {
+            state.transactions = [...state.confirmedTransactions, ...state.unconfirmedTransactions, ...state.partialTransactions];
+        },
+        filterTransactions: (
```

Comment:
The `ADD_TRANSACTION` or `REMOVE_TRANSACTION` actions below send a refresh signal. How does this mutation handle it? I mean if a websocket listener received an unconfirmed/confirmed transaction (there will be a dispatch from the websocket service I think), will that the `transactions` array get updated?

Reply:
done